aiohttp_pydantic/oas/struct.py

```diff
@@ -309,6 +309,10 @@ class Components:
     def __init__(self, spec: dict):
         self._spec = spec.setdefault("components", {})
 
+    @property
+    def examples(self) -> dict:
```

Comment:
Should return an object with attribute summary, description, value ...
https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.0.md#example-object

Reply:
To clarify, you'd like me to add a class abstraction similar to the other structs?

Reply:
I had followed what the `schemas` method was doing. Maybe the usage differs significantly enough or is there a better reaon to make the interface not consistent across them?